content/api/stats.md

```diff
@@ -0,0 +1,167 @@
+  },
+  "assets": [
+    // A list of [asset objects](#assets-objects)
+  ],
+  "chunks": [
+    // A list of [chunk objects](#chunks-objects)
+  ],
+  "modules": [
+    // A list of [module objects](#modules-objects)
+  ],
+  "errors": [
+    // A list of [error strings](#errors-and-warnings)
+  ],
+  "warnings": [
+    // A list of [warning strings](#errors-and-warnings)
+  ]
+}
+```
+
+
+### Asset Objects
+
+Each `assets` object represents an `output` file emitted from the compilation. They all follow the a similar structure:
+
+``` js
+{
+  "chunkNames": [], // The chunks this asset contains ?
+  "chunks": [ 10, 6 ], // The chunk IDs this asset contains ?
+  "emitted": true, // Whether or not the asset made it to the `output` directory ?
+  "name": "10.web.js", // The `output` filename
+  "size": 1058 // The size of the file in bytes ?
+}
+```
+
+
+### Chunk Objects
+
+Each `chunks` object represents a group of modules known as a [chunk](/glossary#chunk). Each object follows the following structure:
+
+``` js
+{
+  "entry": true, // Whether or not it's an entry chunk
+  "files": [
+    // An array of filename strings that contain this chunk (?)
+  ],
+  "filteredModules": 0, // ?
+  "id": 0, // The ID of this chunk
+  "initial": true, // ?
+  "modules": [
+    // A list of [module objects](#modules-objects)
+  ],
+  "names": [
+    // An list of chunk names contained within this chunk (?)
+  ],
+  "origins": [
+    // See the description below...
+  ],
+  "parents": [], // Parent chunk IDs? Names?
+  "rendered": true, // ?
+  "size": 188057 // Chunk size in bytes (?)
+}
+```
+
+The `chunks` object will also contain a list of `origins` describing how the given chunk originated. Each `origins` object follows the following schema:
+
+``` js
+{
+  "loc": "", // Lines of code that generated this chunk (?)
+  "module": "(webpack)\\test\\browsertest\\lib\\index.web.js", // ?
+  "moduleId": 0, // The ID of the module
+  "moduleIdentifier": "(webpack)\\test\\browsertest\\lib\\index.web.js", // ?
+  "moduleName": "./lib/index.web.js", // ?
+  "name": "main", // ?
+  "reasons": [
+    // A list of the same `reasons` found in [module objects](#module-objects)
+  ]
+}
+```
+
+
+### Module Objects
+
+What good would these statistics be without some description of the compiled application's actual modules? Each module in the dependency graph is represented by the following structure:
+
+``` js
+{
+  "assets": [], // A list of [asset objects](#asset-objects) (?)
+  "built": true, // Maybe this means compiled, loaded or output (?)
+  "cacheable": true, // Whether or not this module is cacheable (?)
+  "chunks": [
+    // IDs of chunks that contain this module (?)
+  ],
+  "errors": 0, // Number of errors when resolving or processing the module (?)
+  "failed": false, // Whether or not compilation failed on this module (?)
+  "id": 0, // The ID of the module
+  "identifier": "(webpack)\\test\\browsertest\\lib\\index.web.js", // Another ID ?
```

Comment:
identifier is a unique internal id.

id is the final id that is emitted and exposed.